packages/utils/README.md

```diff
@@ -0,0 +1,24 @@
+[![](https://user-images.githubusercontent.com/39338561/122415864-8d6a7c00-cf88-11eb-846f-a98a936f88da.png)
+](https://kilt.io)
+
+![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)
+
+# KILT Utils
+
+This package provides different generic utility (and stateless) functions.
```

Comment:
Do we really need to specify that they are stateless? What if that changes in the future?

Reply:
The utils package was specifically defined to only contain stateless functions, that's why I wanted to mention it here.

Reply:
Ok did not know the backstory, so that's fine.